dockers/docker-orchagent/kernel_arp_restore.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+function wait_for_intf_up {
+    INTF_NAME=$1
+
+    until [[ `ip link show $INTF_NAME | grep 'state UP'` ]]; do
+        sleep 0.1;
```

Comment:
This is too minute. I would suggest wait 1 sec. Also, if its never up, we need a way to exit the script after say 5mts or so.

Reply:
Ack, will look into a way to timeout